### ISSUE TITLE
Can’t log into NYTimes

### ISSUE THREAD

**Issue by Sdaswani** [mozilla-mobile/fenix#553]
## Steps to reproduce
1. Register for NYTimes on desktop
2. Try to login on Fenix

### Expected behavior
You can log in

### Actual behavior
See screenshot - you see a spinner and then it just ends and then page just sticks around.

### Device information

* Android device: Galaxy S8
* Fenix version: today’s nightly
![4bb41c2c-175d-4634-bb84-964a49efa822](https://user-images.githubusercontent.com/4879954/52904923-5f71a300-31e7-11e9-9426-38ef2e038e10.jpeg)

**Comment by Sdaswani** [comment]
Probably a GV issue?

**Comment by colintheshots** [comment]
It logs in fine with the Google SSO, but if I create an account for a new email, it does not. I can reproduce. Seems like a GV issue to investigate. I created an issue in Bugzilla to start the investigation.

https://bugzilla.mozilla.org/show_bug.cgi?id=1528585

**Comment by colintheshots** [comment]
Actually, this work in Focus 8.0.8 with GeckoView. This seems to be an issue specific to Fenix. We need to investigate this.

**Comment by pocmo** [comment]
I noticed that Fenix has always tracking protection on (I'm not sure how many tracking groups). Maybe that's causing it?

**Comment by boek** [comment]
So this is due to the tracking protection policy. Javascript crashes when it tries to load the captcha.  

```
TypeError: "this.grecaptcha is undefined"
    value React
    value FullPageView.js:148
    value FullPageView.js:35
    React 11
    value FullPageView.js:140
    value Form.js:356
    value Form.js:311
    o Api.js:39
react-dom.production.min.js:179:139
TypeError: "this.grecaptcha is undefined"
    React 11
    value FullPageView.js:140
    value Form.js:356
    value Form.js:311
    o Api.js:39
react-dom.production.min.js:179:139
```

Works as intended when I turn it off

**Comment by boek** [comment]
This issue should be fixed by #1084

**Comment by boek** [comment]
@vesta0 @bbinto I'm going to close this for now. Unless you think further action is needed
